### PR TITLE
internal/cli: add support for removedb

### PR DESCRIPTION
**Pull request by manav2401**
This PR adds support for removedb in the new-cli with similar functionality as in the old cli. 

It can be used as `bor removedb <datadir>` and is responsible to clear the blockchain and state databases. 

Note: It assumes that the `ancient` db lies inside the `chaindata` folder in datadir as we don't have access to the `DatabaseFreezer` path in the code. 

Jira - [POS-699: Support removedb in new-cli](https://polygon.atlassian.net/browse/POS-699)

### CODE REVIEW

File: internal/cli/removedb.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+	// create ethereum node config with just the datadir
+	nodeCfg := &node.Config{DataDir: datadir}
+
+	// Remove the full node state database
+	path := nodeCfg.ResolvePath("chaindata")
+	if common.FileExist(path) {
+		confirmAndRemoveDB(c.UI, path, "full node state database")
+	} else {
+		log.Info("Full node state database missing", "path", path)
```
Comment (JekaMas) [inline internal/cli/removedb.go:84]: we'd like to continue execution in that case?

File: internal/cli/removedb.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	// under the "chaindata" folder.
+	path = filepath.Join(nodeCfg.ResolvePath("chaindata"), "ancient")
+	if common.FileExist(path) {
+		confirmAndRemoveDB(c.UI, path, "full node ancient database")
+	} else {
+		log.Info("Full node ancient database missing", "path", path)
+	}
+
+	// Remove the light node database
+	path = nodeCfg.ResolvePath("lightchaindata")
```
Comment (JekaMas) [inline internal/cli/removedb.go:99]: could you define a bunch of constants for string values?

File: internal/cli/removedb.go [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+			case "y":
+				start := time.Now()
+				_ = filepath.Walk(database, func(path string, info os.FileInfo, err error) error {
+					// If we're at the top level folder, recurse into
+					if path == database {
+						return nil
+					}
+					// Delete all the files, but not subfolders
+					if !info.IsDir() {
+						os.Remove(path)
```
Comment (JekaMas) [inline internal/cli/removedb.go:130]: do we need handle the error here?

**Review by JekaMas — APPROVED** [review]

**Review by pratikpatil024 — APPROVED** [review]

File: internal/cli/removedb.go (hunk 1 above)
Comment (manav2401) [reply internal/cli/removedb.go:84]: Resolving this as per our last conversation.

**Review by cffls — APPROVED** [review]

**Pull request merged**